### ISSUE TITLE
Kernel specs don't appear until I reload the window

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode-jupyter#11815]
Testing #11755

- Run `ipython kernel install --name "venv1-kernel3" --user`
- No new entry in the kernel picker until I reload the window

**Comment by DonJayamanne** [comment]
> No new entry in the kernel picker until I reload the window

Yes, this is a known issue, we don't monitor the these directories for changes nor do we query them from time to time.

**Comment by kebuAAA** [comment]
> > No new entry in the kernel picker until I reload the window
> 
> Yes, this is a known issue, we don't monitor the these directories for changes nor do we query them from time to time.


Maybe I meet the same question, but I can't be quite for sure. Yesterday I update my R and change the path of updated R, but I find that I won't find my updated Irkernel until I open the R ternimal, and the old Irkernel  is not a valid path but it is still in the `select kernel` list.
If this is not a bug, could you give me some help?

**Comment by DonJayamanne** [comment]
> the old Irkernel is not a valid path but it is still in the select kernel list.
If this is not a bug, could you give me some help?
@kebuAAA
please could you file a separate issue

**Comment by kebuAAA** [comment]
> 

OK，I feel so sorry for bothering you

**Comment by DonJayamanne** [comment]
> OK，I feel so sorry for bothering you

please don't apologize, i think you've misunderstood my comment, i do want to help you and an study you're running into this issue.
I'd like to keep your issue separate, else it gets difficult when we have multiple different issues in the same place.
please do file an issue and I'll be glad to help you get  unblocked

**Comment by DonJayamanne** [comment]
Closing for now, now one else has reported this.
